errors/custome_error.js

```
@@ -0,0 +1,20 @@
+
+// custome api error
+// extends from error
+// this is a class
+class CustomeAPIError extends Error {
+    constructor(message) {
+        super(message)
+    }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)